app/workers/update_tx_fee_worker.rb

```diff
@@ -0,0 +1,10 @@
+class UpdateTxFeeWorker
```

Comment:
I don't think that we need a separate worker to update tx fee, we can calculate it in `save_block` method.

Reply:
for calculate tx fee we need previous_cell_output,  but in save block we may not have previous_cell_output. so I use a separate worker to update tx fee.